spec/rails_helper.rb

```diff
@@ -58,6 +58,10 @@
     Rails.root.join('spec/fixtures')
   ]
 
+  config.before(:each) do
+    ActiveJob::Base.queue_adapter = :test
+  end
+
```

Comment:
I think this should be in `environments/test.rb` maybe?  Eg from here: https://api.rubyonrails.org/classes/ActiveJob/QueueAdapters/TestAdapter.html